exchange/exchange.go

```diff
@@ -861,8 +873,9 @@ func errorsToMetric(errs []error) map[metrics.AdapterError]struct{} {
 func errsToBidderErrors(errs []error) []openrtb_ext.ExtBidderMessage {
 	sErr := make([]openrtb_ext.ExtBidderMessage, 0)
 	for _, err := range errortypes.FatalOnly(errs) {
+		errorCode := errortypes.ReadCode(err)
 		newErr := openrtb_ext.ExtBidderMessage{
-			Code:    errortypes.ReadCode(err),
```

Comment:
Why was this change made?

Reply:
Reverted this change. It is unwanted. Initially I was trying to accommodate, seatnonbid related changes here